README.md

```diff
@@ -43,8 +43,7 @@ go get -u github.com/gojektech/heimdall/v6
 This package can be used by adding the following import statement to your `.go` files.
 
 ```go
-import "github.com/gojektech/heimdall/v6/httpclient" // with go modules enabled (GO111MODULE=on or outside GOPATH)
-import "github.com/gojektech/heimdall/httpclient" // with go modules disabled
```

Comment:
Let's add this back? Any reason for us to remove this?

Reply:
With go modules disabled it will be pointing to wrong dependency `gojektech` again. CMIIW.

I am not recommending user to import from previous release than `v7`.